File: backend/src/main/java/com/skybooker/passenger/entity/Passenger.java
package com.skybooker.passenger.entity;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class Passenger {
    private Long id;
    private Long userId;
    private String name;
    private String idCardNo;
    private String passengerType;
    private String phone;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: frontend/src/types/passenger.ts
/** 乘机人 VO（用户保存的常用乘机人） */
export interface PassengerVO {
  id: number
  userId: number
  name: string
  idCardNo: string
  passengerType: "ADULT" | "CHILD" | "INFANT"
  phone: string
}

/** 创建/更新乘机人 DTO */
export interface PassengerDTO {
  name: string
  idCardNo: string
  passengerType: "ADULT" | "CHILD" | "INFANT"
  phone: string
}
